src/routes/getTopUsers.js

```diff
@@ -2,6 +2,7 @@ var db = require('../databases/databases.js').db;
 
 module.exports = function getTopUsers (req, res) {
   let sortType = req.query.sortType;
+  let categoryStatsEnabled = req.query.categoryStats == undefined ? false : true;
```

Comment:
can just be ` let categoryStatsEnabled = req.query.categoryStats` as `undefined` is falsey.